lib/RdfBuilder.js

```diff
@@ -37,6 +66,18 @@ class RdfBuilder extends Readable {
       this.push(rdf.quad(datasetTerm, ns.skos.notation, rdf.literal(subjectCode.values[0])))
     }
 
+    const creationDate = cube.value('CREATION-DATE')
+
+    if (creationDate) {
+      this.push(rdf.quad(datasetTerm, ns.schema.dateCreated, rdf.literal(DateTime.fromFormat(creationDate.values[0], 'yyyyMMdd mm:ss').toISO(), 'http://www.w3.org/2001/XMLSchema#dateTime')))
+    }
+
+    const lastUpdated = cube.value('LAST-UPDATED')
+
+    if (lastUpdated) {
+      this.push(rdf.quad(datasetTerm, ns.schema.dateModified, rdf.literal(DateTime.fromFormat(lastUpdated.values[0], 'yyyyMMdd mm:ss').toISO(), 'http://www.w3.org/2001/XMLSchema#dateTime')))
```

Comment:
Can you change the namespace string to `ns.xsd.dateTime`? The datatype must be provided as a RDF/JS `NamedNode`. The namespace object would handle that also it looks a little bit nicer.